src/pages/workspace/rules/ExpenseReportRulesSection.tsx

```diff
@@ -0,0 +1,193 @@
+import OfflineWithFeedback from '@components/OfflineWithFeedback';
+import Section from '@components/Section';
+import Text from '@components/Text';
+import TextLink from '@components/TextLink';
+import useLocalize from '@hooks/useLocalize';
+import usePolicy from '@hooks/usePolicy';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as CurrencyUtils from '@libs/CurrencyUtils';
+import Navigation from '@libs/Navigation/Navigation';
+import * as PolicyUtils from '@libs/PolicyUtils';
+import ToggleSettingOptionRow from '@pages/workspace/workflows/ToggleSettingsOptionRow';
+import * as PolicyActions from '@userActions/Policy/Policy';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+type ExpenseReportRulesSectionProps = {
+    policyID: string;
+};
+
+function ExpenseReportRulesSection({policyID}: ExpenseReportRulesSectionProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const policy = usePolicy(policyID);
+
+    // Auto-approvals and self-approvals are unavailable due to the policy workflows settings
+    const workflowApprovalsUnavailable = PolicyUtils.getWorkflowApprovalsUnavailable(policy);
+    const autoPayApprovedReportsUnavailable = policy?.reimbursementChoice === CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_NO;
+
+    const renderFallbackSubtitle = (featureName: string) => {
+        return (
+            <Text style={[styles.flexRow, styles.alignItemsCenter, styles.w100, styles.mt2]}>
+                <Text style={[styles.textNormal, styles.colorMuted]}>{translate('workspace.rules.expenseReportRules.unlockFeatureGoToSubtitle')}</Text>{' '}
+                <TextLink
+                    style={styles.link}
+                    onPress={() => Navigation.navigate(ROUTES.WORKSPACE_MORE_FEATURES.getRoute(policyID))}
+                >
+                    {translate('workspace.common.moreFeatures').toLowerCase()}
+                </TextLink>{' '}
+                <Text style={[styles.textNormal, styles.colorMuted]}>{translate('workspace.rules.expenseReportRules.unlockFeatureEnableWorkflowsSubtitle', featureName)}</Text>
+            </Text>
+        );
+    };
+
+    const optionItems = [
+        {
+            title: translate('workspace.rules.expenseReportRules.customReportNamesTitle'),
+            subtitle: translate('workspace.rules.expenseReportRules.customReportNamesSubtitle'),
+            switchAccessibilityLabel: translate('workspace.rules.expenseReportRules.customReportNamesTitle'),
+            isActive: policy?.shouldShowCustomReportTitleOption,
+            pendingAction: policy?.pendingFields?.shouldShowCustomReportTitleOption,
+            onToggle: (isEnabled: boolean) => PolicyActions.enablePolicyDefaultReportTitle(policyID, isEnabled),
+            subMenuItems: [
+                <OfflineWithFeedback pendingAction={policy?.fieldList?.[CONST.POLICY.FIELD_LIST_TITLE_FIELD_ID].pendingFields?.defaultValue}>
```

Comment:
How about this way?

```suggestion
                <OfflineWithFeedback pendingAction={(!policy?.pendingFields?.shouldShowCustomReportTitleOption && policy?.fieldList?.[CONST.POLICY.FIELD_LIST_TITLE_FIELD_ID].pendingFields?.defaultValue) ? policy?.fieldList?.[CONST.POLICY.FIELD_LIST_TITLE_FIELD_ID].pendingFields?.defaultValue) : null 
  }>
```

Reply:
might be confusing after some time, but seems to be working as expected, thank you!

Reply:
@BrtqKr Could you please detail this point? I think we need to discuss about this to avoid regression in the future

> might be confusing after some time